### DISCUSSION TITLE
3.0.0-alpha.15

### DISCUSSION THREAD

**Discussion by tangjinzhou** [vueComponent/ant-design-vue#5016] [General]
## 3.0.0-alpha.15

`2021-12-12`

- 🌟 Optimize Layout performance
- 🌟 Menu supports lazy loading (SubMenu must fill in the key) to improve performance [#4812](https://github.com/vueComponent/ant-design-vue/issues/4812)
- 🌟 Input and Textarea support lazy command modifier [#4951](https://github.com/vueComponent/ant-design-vue/issues/4951)
- 🐞 Select placeholder supports slot [#4995](https://github.com/vueComponent/ant-design-vue/issues/4995)
- 🐞 Fix Radio cursor style [#4997](https://github.com/vueComponent/ant-design-vue/issues/4997)
- 🐞 Fix Statistic.Countdown property support slot [#4996](https://github.com/vueComponent/ant-design-vue/issues/4996)
- 🐞 Fix FormItem name attribute type error [#4998](https://github.com/vueComponent/ant-design-vue/issues/4998)
- 🐞 Fix Menu hidden animation loss problem
- 🐞 Fix FormItem explain style not responding issue [#5004](https://github.com/vueComponent/ant-design-vue/issues/5004)
- 🐞 Fix the problem that Slider tooltip does not display under special conditions
- 🐞 Fix the problem that Dropdown special conditions trigger two click events [#5002](https://github.com/vueComponent/ant-design-vue/issues/5002)
- 🐞 Fix some components reporting errors under SSR, support Nuxt
- 🐞 Fix the problem that the drop-down box component jumps at the edge [#5008](https://github.com/vueComponent/ant-design-vue/issues/5008)
- 🐞 Fix Table type error [#5009](https://github.com/vueComponent/ant-design-vue/issues/5009)

------------------

- 🌟 优化 Layout 性能
- 🌟 Menu 支持懒加载(SubMenu 必须填写 key)，提升性能 [#4812](https://github.com/vueComponent/ant-design-vue/issues/4812)
- 🌟 Input、Textarea 支持 lazy 指令修饰符 [#4951](https://github.com/vueComponent/ant-design-vue/issues/4951)
- 🐞 Select placeholder 支持 slot [#4995](https://github.com/vueComponent/ant-design-vue/issues/4995)
- 🐞 修复 Radio cursor 样式 [#4997](https://github.com/vueComponent/ant-design-vue/issues/4997)
- 🐞 修复 Statistic.Countdown 属性支持插槽 [#4996](https://github.com/vueComponent/ant-design-vue/issues/4996)
- 🐞 修复 FormItem name 属性类型错误 [#4998](https://github.com/vueComponent/ant-design-vue/issues/4998)
- 🐞 修复 Menu 隐藏动画丢失问题
- 🐞 修复 FormItem explain style 未响应问题 [#5004](https://github.com/vueComponent/ant-design-vue/issues/5004)
- 🐞 修复 Slider tooltip 特殊条件不显示问题
- 🐞 修复 Dropdown 特殊条件触发两次 click 事件问题 [#5002](https://github.com/vueComponent/ant-design-vue/issues/5002)
- 🐞 修复部分组件在 SSR 下报错问题，支持 Nuxt
- 🐞 修复下拉框组件，在边缘处位置跳动问题 [#5008](https://github.com/vueComponent/ant-design-vue/issues/5008)
- 🐞 修复 Table 类型错误 [#5009](https://github.com/vueComponent/ant-design-vue/issues/5009)

<hr /><em>This discussion was created from the release <a href='https://github.com/vueComponent/ant-design-vue/releases/tag/3.0.0-alpha.15'>3.0.0-alpha.15</a>.</em>

**Comment by kimmy-wang** [comment]
@tangjinzhou `menu`组件在`ssr`下不显示的问题解决了？
